Modules/System/Time Zone Management/src/DateTimeOffset.Codeunit.al

```diff
@@ -0,0 +1,38 @@
+codeunit 8720 "DateTime Offset"
```

Comment:
As this is a public codeunit you must add:
```
/// <summary>
/// write me
/// <example>
write me
/// </example>
/// </summary>
```

Also, you need to add the open source header to all objects:
```
// ------------------------------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.
// ------------------------------------------------------------------------------------------------
```

Reply:
Added the MIT license header and the summary.